blurb/blurb.py

```diff
@@ -1601,10 +1628,10 @@ def main():
         def handle_option(s, dict):
             name = dict.get(s, None)
             if not name:
-                sys.exit('blurb: Unknown option for {}: "{}"'.format(subcommand, s))
+                sys.exit(f('blurb: Unknown option for {subcommand}: "{s}"'))
```

Comment:
I would probably use `{s!r}` here instead of `"{s}"`, but I realize it's outside the scope of your direct changes.

Reply:
You would change the name of the *variable* from *s* to *str?*

Reply:
No, I would use `!r` to add the quotes, instead of adding them manually. I realize it might change the type of quotes from double to single, based on what `s` contains.

Reply:
Oh!  Sorry.  Tiny font made `s!r` look like `str` on my screen.